io.github.alescdb.mailviewer.json

```diff
@@ -0,0 +1,70 @@
+{
+  "id": "io.github.alescdb.mailviewer",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "47",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.rust-stable"
+  ],
+  "command": "mailviewer",
+  "finish-args": [
+    "--share=network",
+    "--share=ipc",
+    "--socket=fallback-x11",
+    "--device=dri",
+    "--socket=wayland",
+    "--filesystem=home",
+    "--filesystem=xdg-run/mailviewer:create"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin:/usr/lib/sdk/llvm16/bin",
+    "env": {
+      "CARGO_HOME": "/run/build/mailviewer/cargo"
+    }
+  },
+  "cleanup": [
+    "/include",
+    "/lib/pkgconfig",
+    "/man",
+    "/share/doc",
+    "/share/gtk-doc",
+    "/share/man",
+    "/share/pkgconfig",
+    "*.la",
+    "*.a"
+  ],
+  "modules": [
+    {
+      "name": "gmime",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/jstedfast/gmime.git",
+          "tag": "3.2.15"
+        }
+      ]
+    },
+    {
+      "name": "mailviewer",
+      "builddir": true,
+      "buildsystem": "meson",
+      "build-options": {
+        "env": {
+          "CARGO_HOME": "/run/build/mailviewer/cargo",
+          "OFFLINE_BUILD": "1"
+        }
+      },
+      "config-opts": [
+        "--buildtype=release"
+      ],
+      "sources": [
+        "mailviewer-sources.json",
+        {
+          "type": "git",
+          "url": "https://github.com/alescdb/mailviewer.git",
+          "tag": "0.9.5"
```

Comment:
do I understand that corretly? You did retag? Please add a commit here in addition to the tag. This protects against it.

Reply:
I update tag every time I made major changes (so github actions rebuild upstream) or I need to push changes here.
But I didn't delete & repush the tag.

Reply:
yeah I missed the commit in between updating the tag here. For some reason github didn't show it.